content/docs/gke/private-clusters.md

```diff
@@ -48,6 +48,7 @@ from the nodes. Google services (such as BigQuery and Cloud Storage) are still a
 Importantly this means you can continue to use your [Google Container Registry (GCR)](https://cloud.google.com/container-registry/docs/) to host your Docker images. Other Docker registries (for example, DockerHub) will not be accessible. If you need to use Docker images
 hosted outside GCR you can use the scripts provided by Kubeflow to mirror them to your GCR registry.
 
+There is however still a way to pull images directly from DockerHub, or other repositories, and not having the hassle of re-writing deployment files. Activating Google NAT service to the subnet you are using, will give your pods egress access
```

Comment:
@jlewi Please would you review this suggestion?

@oxxygenio We need to give more information in the docs, so that readers know what the proposed solution is, whether it's applicable to them, and how they can use it. For example, you should expand the abbreviation "NAT" (I think it's Cloud NAT?), link to the docs (these? https://cloud.google.com/nat/docs/overview), and give an example to illustrate how people can use this option.

Reply:
Thanks @sarahmaddox and @oxxygenio. 

Should we create a separate/new page that disucsses setting up a Cloud Nat and link to from the private GKE cluster page?

In the Cloud Nat Page it would be nice to have example commands that illustrate how to setup Cloud Nat.

On that page we could then have subsections corresponding to adding access for various services. So adding access to DockerHub would be one (and we could add others later as necessary).

The private GKE page could then just have something along the lines of
Alternatively follow these instructions to setup Cloud Nat to allow access to DockerHub.

Reply:
@jlewi That's a good suggestion. A separate page about Cloud NAT provides more clarity for readers, and gives us more scope to expand the information.

@oxxygenio Are you happy to create such a page?
* Create a new file in this directory: https://github.com/kubeflow/website/tree/master/content/docs/gke
* Copy the "page front matter" (the top lines between the plus signs) from the top of this page:
https://raw.githubusercontent.com/kubeflow/website/master/content/docs/gke/private-clusters.md
* Paste the front matter into your new file, then change the page title etc in the front matter.
* Add your content about Cloud NAT into the same file.
* Add a link from the "private clusters" page pointing to your new page - that is, replace the content you added in this PR with the link instead.

If you do all that in this PR, then you'll be able to view your changes at the same preview link: https://deploy-preview-1563--competent-brattain-de2d6d.netlify.com/docs/gke/private-clusters/

You can also build the site locally and watch your changes appear in a local version of the docs as you type - see the README:
https://github.com/kubeflow/website/blob/master/README.md

Reply:
@jlewi Do you think we should merge this PR and create an issue for the separation of this info into a new page? I don't know whether this content inherently belongs on this page or should be on a separate page from the beginning.